modules/reports/server-ts/excel/helpers/generateExcel.ts

```diff
@@ -0,0 +1,37 @@
+import * as xl from 'excel4node';
+
+const wb = new xl.Workbook();
+const options = {
+  sheetFormat: {
+    defaultColWidth: 25
+  }
+};
+const style = wb.createStyle({
+  font: {
+    color: '#000000',
+    size: 12
+  }
+});
+
+export default function generateBufferExcel(reports: object[]) {
```

Comment:
Please write interface for reports prop.

Reply:
Fixed in #1023